dev-env/bin/daml-sdk-head

```diff
@@ -18,11 +18,15 @@ fi
 
 
 NUKE=0
+PROFILING=0
 for opt in "$@" ; do
   case $opt in
     "--nuke")
       NUKE=1
       ;;
+    "--profiling")
+      PROFILING=1
+      ;;
     *)
       echo "Unknown option: $opt"
       echo "Available options:"
```

Comment:
Please add `--profiling` to the list of available options :-)

Reply:
Good catch! I never realized that `daml-sdk-head` lists available options or tbh I only realized that it had options at all when I added one :slightly_smiling_face: